pkg/virt-operator/resource/generate/rbac/cluster_test.go

```diff
@@ -0,0 +1,315 @@
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesVNCScreenshot), virtv1.SubresourceGroupName, ApiVMInstancesVNCScreenshot, "get"),
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesPortForward), virtv1.SubresourceGroupName, ApiVMInstancesPortForward, "get"),
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesGuestOSInfo), virtv1.SubresourceGroupName, ApiVMInstancesGuestOSInfo, "get"),
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesFileSysList), virtv1.SubresourceGroupName, ApiVMInstancesFileSysList, "get"),
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesUserList), virtv1.SubresourceGroupName, ApiVMInstancesUserList, "get"),
+
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesPause), virtv1.SubresourceGroupName, ApiVMInstancesPause, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesUnpause), virtv1.SubresourceGroupName, ApiVMInstancesUnpause, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesAddVolume), virtv1.SubresourceGroupName, ApiVMInstancesAddVolume, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesRemoveVolume), virtv1.SubresourceGroupName, ApiVMInstancesRemoveVolume, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesFreeze), virtv1.SubresourceGroupName, ApiVMInstancesFreeze, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesUnfreeze), virtv1.SubresourceGroupName, ApiVMInstancesUnfreeze, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMInstancesSoftReboot), virtv1.SubresourceGroupName, ApiVMInstancesSoftReboot, "update"),
+
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMExpandSpec), virtv1.SubresourceGroupName, ApiVMExpandSpec, "get"),
+				Entry(fmt.Sprintf("get %s/%s", virtv1.SubresourceGroupName, ApiVMPortForward), virtv1.SubresourceGroupName, ApiVMPortForward, "get"),
+
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMStart), virtv1.SubresourceGroupName, ApiVMStart, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMRestart), virtv1.SubresourceGroupName, ApiVMStop, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMAddVolume), virtv1.SubresourceGroupName, ApiVMRestart, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMRemoveVolume), virtv1.SubresourceGroupName, ApiVMAddVolume, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMMigrate), virtv1.SubresourceGroupName, ApiVMMigrate, "update"),
+				Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMMemoryDump), virtv1.SubresourceGroupName, ApiVMMemoryDump, "update"),
```

Comment:
Could you add the Entry for the `ApiVMAddInterface`?
```
Entry(fmt.Sprintf("update %s/%s", virtv1.SubresourceGroupName, ApiVMAddInterface), virtv1.SubresourceGroupName, ApiVMAddInterface, "update"),
```

Reply:
good catch!

Reply:
I'm embarassing myself here. Of course, added the test as well.

Reply:
That's fair! Complicated backport ;)

Reply:
@0xFelix cmon, long and boring backports with conflicts can trick anyone! What's important is that we caught and fixed them quickly :+1: